pkg/operator/starter.go

```diff
@@ -117,6 +123,29 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 			"rbac/kube_rbac_proxy_role.yaml",
 			"rbac/kube_rbac_proxy_binding.yaml",
 		},
+	).WithConditionalStaticResourcesController(
+		"AWSEBSDriverConditionalStaticResourcesController",
+		kubeClient,
+		dynamicClient,
+		kubeInformersForNamespaces,
+		assets.ReadFile,
+		[]string{
+			"volumesnapshotclass.yaml",
+		},
+		// Only install when CRD exists.
+		func() bool {
+			name := "volumesnapshotclasses.snapshot.storage.k8s.io"
+			_, err := apiExtClient.ApiextensionsV1().CustomResourceDefinitions().Get(context.TODO(), name, metav1.GetOptions{})
+			if err != nil {
+				klog.Warningf("Not deploying a VolumeSnapshotClass due to error getting CRD: %v", err)
```

Comment:
How often will be this logged?

Reply:
Since this is a separate controller, and there are no watchers for VolumeSnapshotClasses, this will be logged at the controllers' fixed resync interval rate.

Reply:
Seems like by default the StaticResourceController resyncs every minute: https://github.com/bertinatto/aws-ebs-csi-driver-operator-1/blob/650455c6de1a2bd4276c7b6a4f1d986cb9de198d/vendor/github.com/openshift/library-go/pkg/operator/staticresourcecontroller/static_resource_controller.go#L114

Reply:
I assume this is too much. Pushed a change to not log anything. PTAL